Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/NormalizedCPUPercenageGauge.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.WebAppPerformanceCollector
+{
+    using System;
+    using System.Globalization;
+
+    /// <summary>
+    /// Gauge that computes normalized CPU percentage utilized by a process by utilizing the last computed time.
+    /// </summary>
+    internal class NormalizedCPUPercenageGauge : CPUPercenageGauge
+    {
+        /// <summary>Specific environment variable for Azure App Services.</summary>
+        private const string ProcessorsCounterEnvironmentVariable = "NUMBER_OF_PROCESSORS";
```

Comment:
Ideally, just have a gauge that devides value from `ICounterProvider` to whatever you pass in constructor. And pass processor count in constructor with all the checks on being non-negative and not too huge